keras_hub/src/models/roformerV2/roformerV2_text_classifier_preprocessor.py

```diff
@@ -0,0 +1,119 @@
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.bert.bert_text_classifier_preprocessor import (
+    BertTextClassifierPreprocessor,
+)
+from keras_hub.src.models.roformerV2.roformerV2_backbone import (
+    RoformerV2Backbone,
+)
+from keras_hub.src.models.roformerV2.roformerV2_tokenizer import (
+    RoformerV2Tokenizer,
+)
+
+
+@keras_hub_export(
+    [
+        "keras_hub.models.RoformerV2TextClassifierPreprocessor",
+        "keras_hub.models.RoformerV2Preprocessor",
```

Comment:
we can ditch this second export name. we only had it in bert for backwards compat.